### ISSUE TITLE
Export as PDF - landscape not filled

### ISSUE THREAD

**Issue by XXLRay** [bardsoftware/ganttproject#1682]
I can export the project as pdf as landscape but the page is not filled to the right border. It seems as if the portrait export is fitted to the left side of a landscape page.

GanttProject 2.8.9 Pilsen (build 2335)
Windows 10 Pro

**Comment by dbarashev** [comment]
Sorry? What do you mean when you say "filled"? Any example?

**Comment by XXLRay** [comment]
Here is a screenshot:
![landscape not filled](https://user-images.githubusercontent.com/4184977/63671870-e16b0600-c7e0-11e9-8411-12d2425356ce.png)

**Comment by dbarashev** [comment]
Yes, PDF tries to fit the whole chart into the given page size. You have a few options:
1. Zoom in the chart. This will make the chart image wider keeping the same height 
2. Choose some other page size (e.g. A0)
3. If you need it for printing, we recommend exporting to PNG and printing using appropriate tools.

**Comment by bally84** [comment]
when i do png, it prints the list in once big pages, as attached, and when i print, its too small to read?
![Coal Exchange](https://user-images.githubusercontent.com/56552105/66762043-19b3ca00-ee9d-11e9-9550-311c7545e89b.png)

**Comment by dbarashev** [comment]
You can use image printing tools (e.g. Microsoft Paint) which have an option of making a poster (aka tile) from a big image over a few sheets of small paper.

There are online tools as well, e.g. this one comes at the first page of google search and looks nice:: http://posterizer.online/rasterbator/

Here is the (compressed) result for your image
[rasterbation (1)_compressed.pdf](https://github.com/bardsoftware/ganttproject/files/3725152/rasterbation.1._compressed.pdf)

**Comment by bally84** [comment]
ah, great. Thanks
